source/localizable/index.html.erb

```diff
@@ -161,9 +161,53 @@
     </section>
   </div>
 
+  <!---------------------------------- MEDIA ---------------------------------->
+  <div class="body-blackout"></div>
+  <h2 class="text-2xl lg:text-3xl font-extrabold text-center mt-10 m-4" style="color:#F88436"><%= I18n.t('media.title') %></h2>
+  <hr class="border w-12 m-auto mb-8" style="border-color:#F88436">
+  <div class="media-links md:flex w-screen md:justify-center">
+    <div class="media-container my-4 md:my-0 tooltip">
+      <button class="popup-trigger" data-popup-trigger="one">
+        <%= image_tag 'images/shimbun.png', class: 'media-logo h-100 w-100 object-fit', alt: "kyoto shimbun" %>
+      </button>
+        <span class="tooltiptext"><%= I18n.t('media.shimbun') %></span>
+    </div>
+
+    <div class="popup-modal" data-popup-modal="one">
+      <i class="fas fa-2x fa-times text-white bg-primary p-3 popup-modal__close"></i>
+      <%= image_tag 'images/newspaper.png', alt: "kyoto shimbun" %>
+    </div>
+
+    <div class="media-container my-4 md:my-0 tooltip">
+      <button class="popup-trigger" data-popup-trigger="two">
+        <%= image_tag 'images/nhk.png', class: 'media-logo h-100 w-100 object-fit', alt: "nhk" %>
+      </button>
+        <span class="tooltiptext"><%= I18n.t('media.nhk') %></span>
+    </div>
+
+    <div id="yt-modal" class="popup-modal md:overflow-hidden" data-popup-modal="two">
+      <i class="fas fa-2x fa-times text-white bg-primary p-3 popup-modal__close"></i>
+      <section class="embed-container">
+        <div>
+          <iframe id="yt-video" class="mx-auto" width="560" height="315" src="https://www.youtube.com/embed/C19VU5YKnc0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+      </section>
+    </div>
+
+    <div class="media-container my-4 md:my-0 tooltip">
+      <a href="https://www.lewagon.com/blog/alumni-won-japan-hackathon" target="_blank" rel="noopener noreferrer"><%= image_tag 'images/wagon-tokyo.png', class: 'media-logo h-100 w-100 object-fit', alt: "Le Wagon Tokyo" %></a>
```

Comment:
not sure about the `noreferrer` tag - it's for LW, and I wouldn't mind passing them some traffic
also in the path and alt you say LW **Tokyo**, but in the image you say LW **Japan** - please unify

Reply:
Gotcha, I included it because I read it might be good for security, but I don't actually know that much about it, so I can remove it. 

I think I will use Tokyo instead of Japan.

Reply:
Nvm I used "Japan" since I already edited the image.